eth_tester/utils/backend_testing.py

```diff
@@ -862,6 +846,16 @@ def test_receipt_effective_gas_price_for_mined_transaction_max_fee_limit(self, e
         # effective gas price
         assert receipt['effective_gas_price'] == max_fee
 
+    def test_get_transaction_receipt_for_unmined_transaction_raises(self, eth_tester):
```

Comment:
This move successfully corrected git's algorithm and didn't make it look like I re-wrote over this test (saved the old commit message)